graphic/terminal.go

```diff
@@ -0,0 +1,31 @@
+package graphic
+
+import (
+	"os"
+	"strings"
+)
+
+// normalizeTerminal looks for incompatibilities in the terminal configuration
```

Comment:
Can you be more specific with the language in this?

Reply:
Ignore this. I wrote it, forgot about it, came back not remembering I had the comments.

Reply:
I apologize, maybe I had to be a little more specific, it's just that English is not my native language and it's hard for me to write in it.

Reply:
no worries.